aspnet-core/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM mcr.microsoft.com/dotnet/sdk:5.0 AS build
+
+WORKDIR /src
+COPY ["src/AbpCompanyName.AbpProjectName.Web.Host/AbpCompanyName.AbpProjectName.Web.Host.csproj", "src/AbpCompanyName.AbpProjectName.Web.Host/"]
+COPY ["src/AbpCompanyName.AbpProjectName.Web.Core/AbpCompanyName.AbpProjectName.Web.Core.csproj", "src/AbpCompanyName.AbpProjectName.Web.Core/"]
+COPY ["src/AbpCompanyName.AbpProjectName.Application/AbpCompanyName.AbpProjectName.Application.csproj", "src/AbpCompanyName.AbpProjectName.Application/"]
+COPY ["src/AbpCompanyName.AbpProjectName.Core/AbpCompanyName.AbpProjectName.Core.csproj", "src/AbpCompanyName.AbpProjectName.Core/"]
+COPY ["src/AbpCompanyName.AbpProjectName.EntityFrameworkCore/AbpCompanyName.AbpProjectName.EntityFrameworkCore.csproj", "src/AbpCompanyName.AbpProjectName.EntityFrameworkCore/"]
+WORKDIR "/src/src/AbpCompanyName.AbpProjectName.Web.Host"
+RUN dotnet restore 
+
+WORKDIR /src
+COPY ["src/AbpCompanyName.AbpProjectName.Web.Host", "src/AbpCompanyName.AbpProjectName.Web.Host"]
+COPY ["src/AbpCompanyName.AbpProjectName.Web.Core", "src/AbpCompanyName.AbpProjectName.Web.Core"]
+COPY ["src/AbpCompanyName.AbpProjectName.Application", "src/AbpCompanyName.AbpProjectName.Application"]
+COPY ["src/AbpCompanyName.AbpProjectName.Core", "src/AbpCompanyName.AbpProjectName.Core"]
+COPY ["src/AbpCompanyName.AbpProjectName.EntityFrameworkCore", "src/AbpCompanyName.AbpProjectName.EntityFrameworkCore"]
+WORKDIR "/src/src/AbpCompanyName.AbpProjectName.Web.Host"
+RUN dotnet publish -c Release -o /publish --no-restore
+
+FROM mcr.microsoft.com/dotnet/aspnet:5.0
+EXPOSE 44311
```

Comment:
Why expose only 44311 instead of 80 and 443 ports as default? You are also using 9902 port for angular application.

Reply:
Hi @gterdem, 
Regarding the `EXPOSE 44311`:
The exposed ports in the Dockerfile are merely for readability purposes only. and does not affect or constraint the container to expose any port they want.  
That being said, we can remove line 22 and would **not** or add the default port 80. 

However, my bad, I was using the below code on my aspnet-core/src/ServiceBot.Web.Host/appsettings.json file.

`
"Kestrel": {
    "Endpoints": {
      "Http": {
        "Url": "http://*:44311"
      }
    }
  },
` 

Which is changing the default port to 44311, therefore it was right for my implementation, however, you are right. 
The default port is 80 when the Kestrel Endpoint is **not** being changed, so the correct would be  `EXPOSE 80` in there.

Thanks.

Reply:
However, the port is been updated in this file as well:
aspnet-core/docker/ng/docker-compose-updated.yml

Reply:
Afte the update to default port 80. please see the map in this file: https://github.com/EduardoSantanaSeverino/module-zero-core-template/blob/c96ad561f39ee4f0c250106c0688b0d56f3d345c/aspnet-core/docker/ng/docker-compose.yml#L13

Reply:
After the update to default port 80. please see the map in this file: https://github.com/EduardoSantanaSeverino/module-zero-core-template/blob/c96ad561f39ee4f0c250106c0688b0d56f3d345c/aspnet-core/docker/ng/docker-compose.yml#L13

What this is doing is mapping the 9902 outside the container to the 80 inside the container.